server/coordinator.go

```diff
@@ -472,6 +472,32 @@ func (c *coordinator) sendScheduleCommand(region *core.RegionInfo, step schedule
 			},
 		}
 		c.hbStreams.sendMsg(region, cmd)
+	case schedule.AddLearnerPeer:
```

Comment:
Can we remove a learner peer? I believe we need this feature in some cases.

Reply:
I think we can reuse RemovePeer